lib/util/emoji.ts

```diff
@@ -37,6 +43,7 @@ function lazyInitMappings(): void {
     for (const [hex, val] of Object.entries(result.data)) {
       const shortCodes = is.array(val) ? val : [val];
       shortCodesByHex.set(hex, `:${shortCodes[0]}:`);
+      shortCodesByHex.set(stripHexCode(hex), `:${shortCodes[0]}:`);
```

Comment:
what is this doing? 🤔

Reply:
It allows to convert both `26A0-FE0F` (⚠️) and `26A0` (⚠) back to the `:warning:`